src/test/java/Page/LoginPage.java

```diff
@@ -0,0 +1,39 @@
+package Page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class LoginPage extends BasePage{
+
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    public void provideEmail(String email){
+        WebElement emailField = findElement(By.cssSelector("input[type='email']"));
+        emailField.clear();
+        emailField.sendKeys(email);
+    }
+
+    public void providePassword(String password){
+        WebElement passwordField = findElement(By.cssSelector("input[type='password']"));
+        passwordField.clear();
+        passwordField.sendKeys(password);
+    }
+
+    public void clickSubmitBth(){
+        findElement(By.cssSelector("button[type='submit']")).click();
+    }
```

Comment:
We can make the xpath or css the class variable (Locator)

For example

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/d327eccd-5b31-455b-b586-1bbad7d539d5)